Source/dom/components/Artboard.js

```diff
@@ -24,6 +24,15 @@ export class Artboard extends Group {
     }
     super(artboard)
   }
+
+  /**
+   * Adjust the Artboard to fit its children.
+   * override the group's method
+   */
+  adjustToFit() {
+    this._object.resizeToFitChildren()
```

Comment:
is it `resizeToFitChildrenWithOption`?

Reply:
no, `resizeToFitChildrenWithOption` doesn't do anything as you notice. This is a new method that I created

Reply:
this method will not be available until v49.1?

Reply:
no I just created it :)